doc/filesize.rst

```diff
@@ -0,0 +1,72 @@
+The Filesize Extension
+======================
+
+The *Filesize* extension provides the ``filesize`` filter.
+
+You need to register this extension before using the ``filesize`` filter::
+
+    $twig->addExtension(new Twig_Extensions_Extension_Filesize());
+
+``filesize``
```

Comment:
Hmm, how I should manage to get the unit which the user wants? Extra parameter in filter? Or different filters?
If one **unit** extension, would it be better to have an options array as argument instead of a long list of arguments?